client/me/reauth-required/index.jsx

```diff
@@ -111,47 +110,45 @@ const ReauthRequired = createReactClass( {
 				code: this.state.code,
 				remember2fa: this.state.remember2fa,
 			},
-			function ( error, data ) {
+			( error, data ) => {
 				this.setState( { validatingCode: false } );
 				if ( error ) {
 					debug( 'There was an error validating that code: ' + JSON.stringify( error ) );
 				} else {
 					debug( 'The code validated!' + JSON.stringify( data ) );
 				}
-			}.bind( this )
+			}
 		);
-	},
-
-	codeRequestTimer: false,
+	};
 
-	allowSMSRequests: function () {
+	allowSMSRequests() {
 		this.setState( { smsRequestsAllowed: true } );
-	},
+	}
 
-	sendSMSCode: function () {
+	sendSMSCode() {
```

Comment:
Should this be made an arrow function too?

Reply:
That's not needed, because the method is called only once in `handleAuthSwitch` with the `this.sendSMSCode()` convention.